app/models/manageiq/providers/amazon/agent_coordinator.rb

```diff
@@ -200,7 +200,7 @@ def setup_agent(instance)
 
     # docker register
     if docker_login_required?
-      raise "Need credentials to login" unless docker_auth
+      raise("Need credentials to login") unless docker_auth
```

Comment:
I don't think you need this change.

Reply:
Want to use the same format for raising exception in this method.